docs/experimental-features/user-risk-score.asciidoc

```diff
@@ -69,7 +69,7 @@ TIP: If you receive an error message during the installation process, delete the
 [discrete]
 === Upgrade user risk score 
 
-If you previously enabled user risk score and are upgrading to {stack} version 8.5 or later, there'll be an **Upgrade** for user risk score instead of **Enable**.  
+If you previously enabled user risk score and are upgrading to {stack} versions 8.5–8.10, there'll be an **Upgrade** for user risk score instead of **Enable**. If you're upgrading to {stack} version 8.11 or newer, you can upgrade to the latest risk scoring engine.  
```

Comment:
Reference to <<upgrade-risk-engine>> to be added once PR https://github.com/elastic/security-docs/pull/4167 is merged.

Reply:
Same comment as above.